docs/libcudacxx/extended_api/numeric.rst

```diff
@@ -22,3 +24,13 @@ Numeric
      - Casts a value and checks whether the value has changed
      - CCCL 3.1.0
      - CUDA 13.1
+
+   * - :ref:`cuda::overflow_result <libcudacxx-extended-api-numeric-overflow_result>`
+     - Performs addition with overflow checking
```

Comment:
I believe that is not correct, this is the return type of `add_overflow`

Should this be
```suggestion
     - Represent the result of arithmetic operations that may overflow
```

Or do you actually mean `overflow_cast`?